quadrat/sass/theme.scss

```diff
@@ -1,3 +1,3 @@
 @import '../../blank-canvas-blocks/sass/base/breakpoints'; // Get the mobile-only media query and make it available to this theme's styles
-@import 'blocks/media-text';
-@import "utility";
+@import 'blocks/cover';
+@import 'blocks/media-text';@import "utility";
```

Comment:
I guess this shouldn't be on one line?

Reply:
probably from the same rebase that I botched